betatest/templates/betatest/welcome_email.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}
+Hi {{ application.user.profile.full_name }},
+
+we are pleased to inform you that your application to participate in the
+OpenCraft Instance Manager beta test has been accepted.  Your Open edX instance
+has been provisioned and is available at
```

Comment:
@bradenmacdonald Feel free to suggest a better wording for these emails.  As a native speaker, you will probably be able to draft something nicer. :)

(The particular mistake you pointed out here is due to sloppy editing.  It was future before Xavier asked for appservers to be activated immediately, and I changed it to present tense, but missed the "soon".)